src/components/Landing/QAs.tsx

```diff
@@ -46,7 +46,7 @@ export default function QAs() {
           .
         </Trans>,
         <Trans>
-          Projects building on Juicebox pay a 5% JBX membership fee from
+          Projects building on Juicebox pay a 2.5% JBX membership fee from
```

Comment:
**suggestion** can we make the `2.5` a variable? Doing so will mean that the translations won't be erased if we need to change the value in future.

**suggestion (non-blocking)**: Ideally we'll source it from the contract using the `TerminalFee` hook for v1.1, but this could be done in a follow-up (don't want to unnecessarily block the PR).

Reply:
Good call @aeolianeth. Addressed the first point